azure-kusto-ingest/example.js

```diff
@@ -132,34 +58,80 @@ const streamingIngestClient = new StreamingIngestClient(
     props2
 );
 
-// Ingest from file with either file path or FileDescriptor
-streamingIngestClient.ingestFromFile("file.json", null, (err) => {
-    if (err) {
+startStreamingIngestion();
+
+async function startIngestion() {
+    console.log("Ingest from file");
+    try {
+        await ingestClient.ingestFromFile("file.json");
+        console.log("Ingestion done?");
```

Comment:
Should it be removed?

Reply:
I think the original intention was to show we check the status to see if the ingestion done.
It's not related to my change so If you think it's confusing it can be removed...